runtime/queries/vimdoc/injections.scm

```diff
@@ -1,4 +1,4 @@
 ((codeblock
   (language) @injection.language
   (code) @injection.content)
- (#set! injection.include-children))
```

Comment:
That will break the lua->vim injection in our other docs, which I would hate to lose?

Reply:
Are they really worth it?

The alternative (right now) would be to have no injections.

We can re-enable this when we add lazy injections.

Reply:
Yes, they are really worth it. I'd rather drop the comment parser from nvim-treesitter.

Reply:
ok